config/openvpn-client-export/openvpn-client-export.inc

```diff
@@ -807,7 +816,7 @@ function openvpn_client_export_sharedkey_config($srvid, $useaddr, $proxy, $zipco
 			}
 			$conf .= "http-proxy {$proxy['ip']} {$proxy['port']} ";
 		}
-		if ($proxy['proxy_type'] == "socks") 
+		if ($proxy['proxy_type'] == "socks")
```

Comment:
Yes, that should be fine. The pfSense developer style guide is at https://doc.pfsense.org/index.php/Developer_Style_Guide
It specifies no white space at EOL.
And other stuff that is randomly used in code - like right here there are "if(" and "if (" used at random. "if (" is the standard. I am standardizing a lot of the core PHP code at the moment. Maybe there will be a day to do the same to all the package code :)